hangupsbot/plugins/cleverbot.py

```diff
@@ -248,69 +32,25 @@ def _handle_incoming_message(bot, event, command):
     if not bot.get_config_suboption(event.conv_id, 'cleverbot_percentage_replies'):
         return
 
+    if not cw:
+        return
+
     percentage = bot.get_config_suboption(event.conv_id, 'cleverbot_percentage_replies')
 
     if randrange(0, 101, 1) < float(percentage):
-        text = yield from cleverbot_ask(event.conv_id, event.text)
+        text = cw.say(event.text)
```

Comment:
This should call the same function as `chat()` to reduce code repetition. You can either abstract that out into a _chat() function, or call the chat function directly from this function